msteams-platform/concepts/deploy-and-publish/appsource/prepare/teams-store-validation-guidelines.md

```diff
@@ -469,7 +469,7 @@ App packages must be correctly formatted and include all required information an
 >   * **Clearly describe limitations**, conditions, or exceptions to the functionality, features, and deliverables in the app long description and related materials.
 >   * **Emphasis on any considerations** for testers while validating your app submission.
 >   * **Prepopulate the test accounts with dummy data** to aid testing.
->   * If you are providing your test accounts, ensure that you enable third-party integration. Also, disable two-factor or multi-factor authentication.
```

Comment:
What's the way forward if MFA is not disabled? While testing, how can app validation team use the test account?

Reply:
It will be done on call but we are not documenting it on purpose.

Reply:
@Wajeed-msft: @SurbhiJainMSFT has responded that disabling MFA is no longer part of the new SOP. So, we can remove this content from the sections.